libcaliptra/examples/generic/test.c

```diff
@@ -37,6 +37,23 @@ void hwmod_init(struct caliptra_buffer rom, const test_info *info);
 static int dump_array_to_file(struct caliptra_buffer* buffer, const char *filename);
 #endif
 
+#define DPE_DERIVE_CONTEXT_INIT(x)                        \
```

Comment:
I think I have a minor preference for helper functions rather than macros. I think it makes the code a bit easier to follow. For example:

```c
int invoke_dummy_derive_context(uint32_t flags, caliptra_invoke_dpe_resp *resp) {
  ...
}
```

Reply:
No problem, I'll make that change.